Source/m_menu.c

```diff
@@ -6358,6 +6361,27 @@ void M_Init(void)
         strcpy(OptionsMenu[scrnsize].name, "M_DISP");
     }
   }
+
+  // [crispy] remove DOS reference from the game quit confirmation dialogs
+  {
+    const char *platform = SDL_GetPlatform();
+    const char *string;
+    char *replace;
+
+    string = endmsg[3];
+    replace = M_StringReplace(string, "dos", platform);
+    endmsg[3] = replace;
+
+    string = endmsg[4];
+    replace = M_StringReplace(string, "dos", platform);
+    endmsg[4] = replace;
+
+    string = endmsg[9];
+    replace = M_StringReplace(string, "dos", platform);
+    string = M_StringReplace(replace, "prompt", "desktop");
```

Comment:
I could even check for `isatty(fileno(stdout))` on non-Windows and replace "desktop" with "shell" or similar, but well...